utils_cv/detection/helper.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import List, Union
+
+
+class Bbox:
+    """ Util to represent bounding boxes
+
+    Source:
+    https://github.com/Azure/ObjectDetectionUsingCntk/blob/master/helpers.py
```

Comment:
Feel free to edit/remove content etc as you see fit.

Reply:
yep, made some updates, added a few functions, hesitant to delete the un-used ones at this moment thought because they look useful. only a few anyway.